build/common.gni

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains common build flags for building Crosswalk. Users should
+# NOT directly import this file into their build folder.
+# See //xwalk/build/linux.gni for how to import flags for Crosswalk-Linux Build
+
+# Crosswalk specific build flags are defined in xwalk.gni. Import them.
+import("//xwalk/build/xwalk.gni")
+
+root_extra_deps = [ "//xwalk:xwalk_builder" ]
+
+# From src/build/config/features.gni
+enable_nacl = true
+
+# From src/build/config/features.gni
+enable_webvr = false
+
+# From src/build/config/compiler/BUILD.gn
+# If we want stack unwind support for backtrace()
+exclude_unwind_tables = false
+
+# From src/third_party/ffmpeg/ffmpeg_options.gni
+ffmpeg_branding = "Chromium"
+
+# From src/build/config/features.gni
+proprietary_codecs = true
+
+# From src/build_overrides/v8.gni
+v8_use_external_startup_data = true
```

Comment:
The default is already true (and we're only changing it for Android), so you can remove it from here.